driver/aws/datalake_api.py

```diff
@@ -22,42 +22,30 @@ def __init__(self, path_key):
             o = Partition(sub_partition)
             self.subpartitions.append(o)
 
-    def get_partition_chain(
-        self, prefix: str, parent_key: str = None, parent_value: str = None
-    ) -> List[Dict[str, str]]:
+    def get_partition_chain(self, prefix: str, parent_key: str = None, parent_value: str = None) -> List[Dict[str, str]]:
         pchain = list()
-        prepped_prefix = os.path.join(prefix, f"{self.name}={self.value}")
+        prepped_prefix = os.path.join(prefix, f'{self.name}={self.value}')
         pkeys = list()
         pkey_values = list()
         if parent_key and parent_value:
             pkeys.append(parent_key)
             pkey_values.append(parent_value)
         if len(self.subpartitions) > 0:
             for sp in self.subpartitions:
-                pchain.extend(
-                    sp.get_partition_chain(
-                        prepped_prefix, parent_key=self.name, parent_value=self.value
-                    )
-                )
+                pchain.extend(sp.get_partition_chain(prepped_prefix, parent_key=self.name, parent_value=self.value))
         else:
             pkeys.append(self.name)
             pkey_values.append(self.value)
-            pchain.append(
-                {"keys": pkeys, "values": pkey_values, "location": prepped_prefix}
-            )
+            pchain.append({'keys': pkeys, 'values': pkey_values, 'location': prepped_prefix})
         return pchain
 
 
 def read_partitions(bucket: str, container_folder: str = None):
     s3 = providers.get_s3()
-    rsp = s3.list_objects_v2(
-        Bucket=bucket,
-        Prefix=os.path.join(container_folder, ""),
-        ExpectedBucketOwner=providers.get_aws_account_id(),
-    )
-    keys = set(os.path.dirname(k.get("Key")) for k in rsp.get("Contents"))
-    prefix = rsp.get("Prefix")
-    partition_keys = [p[len(prefix) :] for p in keys if p != prefix.rstrip("/")]
+    rsp = s3.list_objects_v2(Bucket=bucket, Prefix=os.path.join(container_folder, ''))
```

Comment:
`providers.get_aws_account_id()` is gone. If this is intended, I am okay with it. Just FYI.